### ISSUE TITLE
More plant actions

### ISSUE THREAD

**Issue by matthijsthoolen** [morkro/happy-plants#96]
In v2 you added a schedule for watering. That would surely help me a lot :) 

Can this schedule be expanded with other scheduled tasks. Would be nice if custom recurring or even one-time events can be added. Events which I can think of now:
- Fertilizer (can be a default one)
- Repotting
- Plant cuttings
- Plant pruning
- Spraying the plants
Probably a lot more can be added, so custom events would be really cool.

That was the last one for now :)

**Comment by morkro** [comment]
Yes, this is what I was thinking to enhance the feature with as well. Great ideas, keep them coming! ;) 

I think for the MVP version of this I will keep it to watering schedules/reminders and once that functionality is implemented/released, it can be extended to more use cases.
